chore: Fix Storybook on CI

rafaeelaudibert opened the pull request:
## Problem

Storybook is broken

## Changes

Fix it

mariusandra reviewed (approved):
Overall, yes, thank you! However any way to not have 1128 changed files :eyes_shaking: :D

rafaeelaudibert commented:
@mariusandra Oh yeah, it's better now, that was just me banging my head to the wall at 2 AM last night, it's better now
![image](https://github.com/user-attachments/assets/34ab9fcb-8c88-42d9-97d7-1385e4f352c9)

rafaeelaudibert commented:
@mariusandra Well, that didn't work, now no files have been committed... I will ping you once this is actually working

rafaeelaudibert committed:
fix: Reinstate snapshots on CI

Our paths were all messed up, let's fix that. I've also added some better debugging tooling to the job which should hopefully make debugging this easier in the future

github-actions[bot] committed:
Update UI snapshots for `webkit` (2)

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

rafaeelaudibert commented:
@mariusandra Working now, will merge it and let teams know about the regressions that this PR found

rafaeelaudibert committed:
fix: Use proper timeout

mariusandra commented on frontend/__snapshots__/components-playerinspector--default--dark.png:
This doesn't look good

rafaeelaudibert replied:
yeah, some don't look good at all, so that's why I'll tag teams in #tell-posthog-anything once this is finally merged, just getting it to stop to flap

rafaeelaudibert committed:
chore: Run 3 storybook shards

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)

github-actions[bot] committed:
Update UI snapshots for `chromium` (3)

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)
